inc/spbc-settings.php

```diff
@@ -3648,7 +3661,11 @@ function spbc_field_scanner__show_accordion($direct_call = false)
     }
 
     if ($spbc->settings['scanner__list_approved_by_cleantalk']) {
-        $tables_files['approved_by_cloud'] = __('Approved by CleanTalk Team or Clout files list. To disable this list view, please disable the `Show approved by CleanTalk Cloud` option.', 'security-malware-firewall');
+        $company_name = 'ClaenTalk';
```

Comment:
First one - there is an typo. Second one - please, get company name by default from [State](https://github.com/CleanTalk/security-malware-firewall/blob/master/lib/CleantalkSP/SpbctWP/State.php#L211)